bridge/src/bridge/gas_price.rs

```diff
@@ -12,7 +12,6 @@ use config::{GasPriceSpeed, Node};
 use error::Error;
 
 const CACHE_TIMEOUT_DURATION: Duration = Duration::from_secs(5 * 60);
```

Comment:
Does it make sense to have `CACHE_TIMEOUT_DURATION` changeable through the configuration file?

Reply:
It would be easy enough to add it to the config file, the only problem that I could see is that if a user accidentally set their cache-duration too low, they could possibly DOS the gas price oracle.

The duration of the cache should reflect Ethereum's gas-price volatility. For example, if the gas-price was rapidly changing, you would want a shorter cache duration, however, looking at the current sate of the Ethereum network, the average gas-price across all confirmed transactions [stays within the [5, 25] GWEI range](https://etherscan.io/chart/gasprice), and has been that way for quite some time. To me, this indicates that cache duration should be a constant.

What are your thoughts?